src/form.js

```diff
@@ -14,18 +16,76 @@ function Form({ label, placeholder, saveName }) {
   }
 
   return (
-    <form onSubmit={setName}>
-      <label>{label}</label>
-      <input
-        required="required"
-        autoComplete="off"
-        name="name"
-        placeholder={placeholder}
-        onChange={onChangeInputName}
-        value={currentValue}
-      />
-      <button type="submit">Next</button>
-    </form>
+    <div
+      css={{
+        height: "100vh",
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center"
+      }}
+    >
+      <form
+        onSubmit={setName}
+        css={{
+          margin: "auto",
+          backgroundColor: "white",
+          borderRadius: ".5rem",
+          padding: "2rem 1rem",
+          fontSize: "25px",
```

Comment:
If you're using rem as units. You should try to use it here.

Reply:
yeah, you're right, I forgot to change it